### DISCUSSION TITLE
Is there a possibility to replace the face in 2 photos that I choose?

### DISCUSSION THREAD

**Discussion by Alessandro891** [lllyasviel/Fooocus#2435] [Q&A] [answered]
For example, I have photo 1 of a person at the bar with a cup of coffee nearby, and 1 photo with the face of another person, I would like that in photo 1 without being manipulated into other shapes, only the face of photo 2 is replaced in real quality. if I use swap face it completely modifies the images and creates a mix between photo 1 and photo 2. Thanks

**Comment by ILikeToasters** [comment, marked as answer]
If I understand correctly you just want to take the face from one image and use it in the other? If so the one you want to change you would put in the inpainting area and mask the face, then the other put in image prompt with Faceswap enabled and in the debug tools you would enable using image prompts with inpainting. But if using regular inpainting you would also need to turn up "Forced Overwrite of Refiner Switch Step" to 50 or so if using the quality setting for example or else it may change the face too much in the last steps. If using the improve detail part of inpainting you do not need to change that setting. Hopefully that is clear. Also I find Faceswap works best with a weight of .85 or higher.

**Reply by Amit30swgoh** [reply]
Thanks, friend!
![image](https://github.com/lllyasviel/Fooocus/assets/42430868/650be549-0045-468d-82cf-7f6d6f456dcc)

**Reply by Alessandro891** [reply]
fantastic it works thank you very much :)
